lamindb/curators/core.py

```diff
@@ -843,7 +912,18 @@ def values(self, new_values):
     @property
     def is_validated(self) -> bool:
         """Return whether the column is validated."""
-        return len(self._non_validated) == 0
+        # ensembl gene IDs pass even if they were not validated
+        # this is a simple solution to the ensembl gene version problem
+        if self._field.field.attname == "ensembl_gene_id":
+            # if none of the ensembl gene ids were validated, we are probably not looking at ensembl gene IDs
+            if len(self.values) == len(self._non_validated):
```

Comment:
I think we should rather make this the general behavior than only having it for ensembl_gene_id.

If there is not a single valid category, likely the dtype is wrong. We could introduce a setting `n_minimal_valid_values=1` or something like this on `CatVector` with _rough_ analogy to `minimal_set` in `Schema` (where we require defined features to be present, not just any valid values).

For now, we don't need to introduce this setting explicitly, but we could simply assume it. I think that if you validate a categorical and there is not a single valid thing in there, you probably have the wrong dtype.

Of course there will be edge cases with this but doing something only for Ensembl Gene IDs will also produce edge cases.